### PR TITLE
Option -C to keep system php.ini

### PR DESCRIPTION
**Pull request by milo**
@dg I'm aware of your propose of `-c default` but `-C` seems less magic to me. Anyway, if you think that saving one new option is better, I'm ok with it (maybe as `-c system`). See #331

### CODE REVIEW

File: src/Runner/CliTester.php [hunk 1]
```diff
@@ -102,6 +102,7 @@ private function loadOptions()
 
 Options:
     -p <path>                    Specify PHP interpreter to run (default: php).
+    -C                           Do not disable system php.ini.
```
Comment (dg) [inline src/Runner/CliTester.php]: `Use system php.ini`? Or `default`?

File: src/Runner/PhpInterpreter.php [hunk 2]
```diff
@@ -28,7 +28,7 @@ class PhpInterpreter
 	private $error;
 
 
-	public function __construct($path, array $args = [])
+	public function __construct($path, $keepSystemIni = FALSE, array $args = [])
```
Comment (dg) [inline src/Runner/PhpInterpreter.php]: When all arguments are passed low-level in `$args`, the  `-n` should be part of `$args` too, or not?

**Comment by dg** [comment]
I am fine with `-C`.

**Comment by milo** [comment]
@dg Thanks for review! Updated.

**Pull request merged**